extensions/wikia/AdEngine/js/tracking/adInfoTracker.js

```diff
@@ -39,7 +40,8 @@ define('ext.wikia.adEngine.tracking.adInfoTracker',  [
 			'browser': [ browserDetect.getOS(), browserDetect.getBrowser() ].join(' '),
 			'country': pageParams.geo || '',
 			'time_bucket': now.getHours(),
-			'timestamp': now.getTime(),
+			'timestamp': timestamp,
+			'ad_load_time': timestamp - window.performance.timing.connectStart,
```

Comment:
Use `win.performance.timing.connectStart` instead in order to keep the global stuff away ;)